hr_course/models/hr_course.py

```diff
@@ -180,6 +183,44 @@ def cancel_course(self):
         for record in self:
             record.write(record._cancel_course_values())
 
+    @api.model
+    def send_email(self, partner):
+        email = partner.email
+        if email and email.strip():
+            email_template = self.env.ref(
+                "hr_course.mail_template_validity_reminder_en"
+            )
+            options = {
+                "email_to": email,
+                "email_from": self.env["res.company"].search([]).email.strip(),
+            }
+            if email_template:
+                email_template.send_mail(self.id, email_values=options, force_send=True)
+                return True
+        raise UserError(
+            _(
+                """Could not send mail to partner %s because it does not\
+                have any email address defined""",
+                partner.display_name,
+            )
+        )
+
+    @api.model
+    def process_validity(self):
+        alerting_delay = self.env["res.company"].search([]).alerting_delay
+        mailing_list = self.env["res.company"].search([]).mailing_list_to_alert
+        for course in self:
+            if course.validity_end_date:
+                if course.alerted is False and course.validity_end_date >= (
```

Comment:
```suggestion
                if  course.validity_end_date >= (
```
already selected in _cron_check_validity_date